docs/development.md

```diff
@@ -149,27 +151,42 @@ make delete-workload-cluster
 Your environment must have the Azure credentials as outlined in the [getting
 started prerequisites section](./getting-started.md#Prerequisites)
 
-#### Building and pushing dev images
+#### Creating a dev cluster
 
-1. Login to your container registry using `docker login`.
+The steps below are provided in a convenient script in [hack/manual-testing.sh](hack/manual-testing.sh). Be sure to set `REGISTRY="<container-registry>"`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the cluster name:
```

Comment:
```suggestion
The steps below are provided in a convenient script in [hack/manual-testing.sh](hack/manual-testing.sh). Be sure to set `REGISTRY="<container-registry>"`, `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
```

It might be me but I get confused with all the different clusters that are used 🤷‍♂